content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.overlay()'
+Description: 'Layers the specified views in front of a view.'
+Subjects:
+  - 'Mobile Development
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.overlay()`** layers the views that you specify in front of a [`View`](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
```

Comment:
Change this pseudo block so that everything is not shifted to the center. Keep left-aligned and use two spaces when indenting instead of 4.